packages/deployer/tasks/deploy.js

```diff
@@ -49,7 +48,6 @@ task(TASK_DEPLOY, 'Deploys all system modules')
 
     await hre.run(SUBTASK_PREPARE_DEPLOYMENT, taskArguments);
     await hre.run(SUBTASK_PRINT_INFO, taskArguments);
-    await hre.run(TASK_COMPILE, { force: true, quiet: true });
```

Comment:
Why is the first compilation moving to `sync-sources.js`?

Reply:
+1 I'd prefer this task to be visible in the deployment process here

Reply:
Did it because the deploy-router was also doing it on the subtask. Moving both compiles to the root :)